spec/models/service_spec.rb

```diff
@@ -53,4 +53,52 @@
       expect(service.required_tiers("all")).to eq([2, 3, 1])
     end
   end
+
+  describe "#duplicate!" do
+    it "duplicates the service" do
+      service = create(:service, :all_tiers, lgsl_code: 1)
+
+      interaction1 = create(:interaction, lgil_code: 1, label: "interaction1")
```

Comment:
I don't think you need to create this, one would be implicitly created when you create a service_interaction

Reply:
Thanks - fixed up as per your suggestion